topcoffea/json/lumi.json

```diff
@@ -1,5 +1,6 @@
 {
- "2016": 36.33,
+ "2016APV": 19.52,
+ "2016": 16.81,
```

Comment:
I think it would be useful to add a line to the [readme](https://github.com/TopEFT/topcoffea/tree/codecov-test2/topcoffea/data/goldenJsons) for the golden json directory to mention the twiki where these 2016 lumi numbers come from (if it's different from the twiki already listed there, which is where the other years' numbers come from).